### PR TITLE
minify_exports

### PR DESCRIPTION
**Pull request by juj**
Adjust MinifyImportsAndExports to optionally minify the export names, sometimes that is not desirable.

### CODE REVIEW

File: src/passes/MinifyImportsAndExports.cpp [hunk 1]
```diff
@@ -150,20 +151,34 @@ struct MinifyImportsAndExports : public Pass {
...
     }
     module->updateMaps();
     // Emit the mapping.
     for (auto& pair : oldToNew) {
       std::cout << pair.second.str << " => " << pair.first.str << '\n';
     }
   }
+
+public:
+  explicit MinifyImportsAndExports(bool minifyExports)
```
Comment (kripken) [inline src/passes/MinifyImportsAndExports.cpp]: for consistency with code style elsewhere, please put this right after the new bool that was added at the top, and format it as
```
MinifyImportsAndExports(bool minifyExports) : minifyExports(minifyExports) {}
```

**Comment by kripken** [comment]
Oh, also please add a test - a new files in `tests/passes/minify-imports.wast` (and a corresponding `.txt` output file, either generate manually or by running `./auto_update_tests.py`)

**Comment by kripken** [comment]
Odd about those unrelated errors. Does `./check.py` pass for you? Maybe our tests just don't run on windows, we only run them on linux here...

**Comment by juj** [comment]
> Does `./check.py` pass for you?

Unfortunately no, that gets the error listed in the last block above. (translate-to-fuzz fails)

**Pull request merged**